api/v1alpha1/k8sgpt_types.go

```diff
@@ -28,20 +28,33 @@ type SecretRef struct {
 	Key  string `json:"key,omitempty"`
 }
 
+type CredentialsRef struct {
```

Comment:
We don't need this type, we can use the existing `v1.Secret` inside of `RemoteCacheRef`

Reply:
Same as the previous comment. Shouldn't we be consistent with the rest of the code and use v1.Secret to manage secrets related to OpenAI credentials? 

```yaml
type SecretRef struct {
	Name string `json:"name,omitempty"`
	Key  string `json:"key,omitempty"`
}
```

Reply:
In `SecretRef`, we attempt to have a name and key of the secret. In the case of `CredentialRef`, we attempt to have `AccessKeyID` ( which we can associate to `SecretRef` `key` attribute ) but we have `SecretAccessKey` and I think we should use two different type to not compexicate the code. Do you have another idea ?